### PR TITLE
Fix block in search by preventing non number

### PR DESCRIPTION
**Pull request by evilpeach**

### CODE REVIEW

**Review by Poafs1 — APPROVED** [review]

File: src/lib/layout/Searchbar.tsx [hunk 1]
```diff
@@ -131,6 +132,18 @@ const ResultItem = ({
...
+    if (isAccountAddress) {
+      return metadata.icns.address || metadata.initiaUsername.address || value;
+    }
+    return value;
+  }, [
+    isAccountAddress,
+    metadata.icns.address,
+    metadata.initiaUsername.address,
+    value,
+  ]);
```
Comment (songwongtp) [inline src/lib/layout/Searchbar.tsx:146]: is this change necessary? i believe we can assign names to contract/SDK module account

**Review by songwongtp — APPROVED** [review]
